app/js/functions.js

```diff
@@ -185,20 +264,15 @@ function parseHash(fieldsById) {
                 .attr("disabled", null);
     */
     deferredUpdate();
-    location.replace("#" + field.id);
+  location.replace("#" + field.id);
 
-    hashish.attr("href", function(href) {
-      return href + location.hash;
-    });
-  }
+  hashish.attr("href", function(href) {
+    return href + location.hash;
+  });
+}
 
 //Inital map setup
-var map = d3.select("#map"),
-  zoom = d3.behavior.zoom()
```

Comment:
Not sure why this looks like this in the pull request but it the stuff is back in if you check the branch source.